core/metacling/test/TClingDataMemberInfoTests.cxx

```diff
@@ -220,3 +227,14 @@ class Outer {
    auto *dmInnerProtected = (TDataMember*)TClass::GetClass("DMLookup::Outer")->GetListOfDataMembers()->FindObject("InnerProtected");
    EXPECT_EQ(dmInnerProtected->Property(), kIsProtected | kIsClass);
 }
+
+#ifdef HAVE_RBOX
+TEST(TClingDataMemberInfo, issue8553)
+{
+   TClass *clBox = nullptr;
+   ASSERT_TRUE(clBox = TClass::GetClass("ROOT::Experimental::RBox"));
+   TDataMember *dmAttrBorder = nullptr;
+   ASSERT_TRUE(dmAttrBorder = clBox->GetDataMember("fAttrBorder"));
```

Comment:
After https://github.com/root-project/root/pull/8566 member will be renamed `fAttrBorder` -> `border`.

Reply:
Well I suppose you need this PR for #8566 so you'd have to rename the member in this test in your PR? ;-) Sorry!

Reply:
No problem, but you have to merge it first